### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by andrew17752985**
https://andrew17752985.github.io/react_tabs-js/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  const handleTabSelected = (tabId) => {
+    if (tabId !== selectedTabId) {
+      setSelectedTabId(tabId);
+    }
```
Comment (maxim2310) [inline src/App.jsx:19]: you need to to this condition in tabs component

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just one tiny thing needs to be fixed before approval)

File: src/App.jsx [hunk 2]
```diff
@@ -1,40 +1,36 @@
...
+  };
```
Comment (TarasHoliuk) [inline src/App.jsx:20]: you should handle this:
image

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
image
try to fix the tests or feel free to ask in the chat

**Review by volodymyr-soltys97 — APPROVED** [review]

**Pull request closed without merging**